robocopkaka/reminder_app
# Add navbar

#### What does this PR do?
Adds a navbar and a background image for the home page
#### Description of Task to be completed?
  - add spec for navbar
  - add background image for home page
  - modify height and max-height values to accommodate navbar
  - add .simplecov file
#### How should this be manually tested?
* Start your server
* Visit `http://localhost:3000`
* You should see a navbar
* You should see authentication buttons
* If you're signed in, you should see buttons for adding and viewing reminders
#### Any background context you want to provide?
N/A
